build_runner_core/tool/build.dart

```diff
@@ -15,11 +16,14 @@ main() async {
     options,
     environment,
     [
-      applyToRoot(jsonSerializable(BuilderOptions.forRoot),
-          hideOutput: false,
+      applyToRoot(
+          jsonSerializable(
+              new BuilderOptions({'generate_to_json_function': false})),
```

Comment:
Turns out the mixin model works well here CC @jakemac53 – or at least I couldn't figure out how to use the functions in a sane way...

Reply:
I was able to migrate to the functions using the normal patterns, pushed a commit here.